integr8ly/installation
# Middleware monitoring config

## Additional Information
Ticket: https://issues.jboss.org/browse/INTLY-666

This should expose the 3scale metrics to the middleware-monitoring stack using [this method](https://issues.jboss.org/browse/INTLY-718?focusedCommentId=13697604&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-13697604).
## Verification Steps

1. Run this installation branch.
2. Go to the Prometheus UI. (`prometheus-route`/config  in the `middleware-monitoring` namespace) and make sure a new config is present.

![config](https://user-images.githubusercontent.com/22113654/53110689-74c23680-3533-11e9-8c81-00265baf9d3d.png)

3. Ensure there are 2 new targets at `prometheus-route`/targets
![targets](https://user-images.githubusercontent.com/22113654/53110501-f36aa400-3532-11e9-88cd-9ccd02c1ebe6.png)
4. Ensure you can run a query v the new targets. e.g. `nginx_http_connections{_3scale_component_element="staging"}` at `prometheus-route`/graph
![graph](https://user-images.githubusercontent.com/22113654/53110622-417fa780-3533-11e9-98f1-649621804ca0.png)
